hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/throttling/GasLimitThrottlingSuite.java

```diff
@@ -106,10 +114,12 @@ private HapiSpec txOverGasLimitThrottled() {
                         contractCall(
                                         CONTRACT,
                                         "twoSSTOREs",
-                                        Bytes.fromHexString("0x05").toArray())
-                                .gas(1_000_001)
+                                        Bytes.fromHexString(
+                                                        "0x0000000000000000000000000000000000000000000000000000000000000005")
+                                                .toArray())
+                                .gas(MAX_GAS_PER_SECOND + 1L)
                                 .payingWith(PAYER_ACCOUNT)
-                                .hasPrecheck(BUSY),
+                                .hasPrecheckFrom(MAX_GAS_LIMIT_EXCEEDED, BUSY),
```

Comment:
Won't the _only_ response be `MAX_GAS_LIMIT_EXCEEDED`?

Reply:
No, in modularized we do the frontend throttles check first so we'll get `BUSY` but in mono we do the semantic check before the throttle one, so there we'll get `MAX_GAS_LIMIT_EXCEEDED`.

Unless we call `dispatcher.dispatchPureChecks()` during ingest in modularized we'll be getting `BUSY` there, but as per @netopyr's comment from above we don't want to do that.